cfme/configure/settings.py

```diff
@@ -20,112 +22,191 @@ class TimeProfileAddForm(View):
     timezone = BootstrapSelect('profile_tz')
     days = BootstrapSwitch(name='all_days')
     hours = BootstrapSwitch(name='all_hours')
-    save_button = Button(VersionPick({Version.lowest(): 'Save', '5.9': 'Add'}))
-    configuration = Dropdown('Configuration')
-    table = Table("//div[@id='main_div']//table")
-    save_edit_button = Button('Save')
+    add = Button(VersionPick({Version.lowest(): 'Save', '5.9': 'Add'}))
     cancel_button = Button('Cancel')
     help_block = Text("//span[contains(@class, 'help-block')]")
 
 
-class TimeprofileAddEntities(View):
+class TimeProfileEntities(View):
+    table = Table("//div[@id='main_div']//table")
     breadcrumb = BreadCrumb()
     title = Text('//div[@id="main-content"]//h3')
 
 
-class TimeProfileAddFormView(BaseLoggedInPage):
-    timeprofile_form = View.nested(TimeProfileAddForm)
-    entities = View.nested(TimeprofileAddEntities)
+class TimeProfileView(BaseLoggedInPage):
+    form = View.nested(TimeProfileAddForm)
+    entities = View.nested(TimeProfileEntities)
     mysetting = View.nested(MySettingsView)
+    configuration = Dropdown('Configuration')
 
+    @property
+    def is_displayed(self):
+        return True if self.entities.title == 'Time Profile Information' else False
 
-class Timeprofile(Updateable, Navigatable):
-    def __init__(self, description=None, scope=None, days=None, hours=None, timezone=None,
-                 appliance=None):
-        Navigatable.__init__(self, appliance=appliance)
-        self.description = description
-        self.scope = scope
-        self.days = days
-        self.hours = hours
-        self.timezone = timezone
 
-    def create(self, cancel=False):
-        view = navigate_to(self, 'All')
-        view.timeprofile_form.configuration.item_select('Add a new Time Profile')
-        view.timeprofile_form.fill({
-            'description': self.description,
-            'scope': self.scope,
-            'days': self.days,
-            'hours': self.hours,
-            'timezone': self.timezone,
-        })
-        if not cancel:
-            view.timeprofile_form.save_button.click()
-            view.flash.assert_message('Time Profile "{}" was saved'.format(self.description))
+class TimeProfileAllView(TimeProfileView):
+
+    @property
+    def is_displayed(self):
+        return self.configuration.is_enabled
```

Comment:
**Required**

This isn't a sufficient is_displayed, a 'Configuration' dropdown widget will match on multiple pages. Need an additional title, breadcrumb, etc check, or return `False` if there isn't anything else to validate against.